int32 traction_position
int32 traction_velocity
int32 steer_position
int32 steer_velocity